src/components/Exchange/PositionSeller.js

```diff
@@ -1034,7 +1053,7 @@ export default function PositionSeller(props) {
             </div>
             <div className="Exchange-info-row">
               <div className="Exchange-info-label">
-                <Trans>Collateral</Trans>
+                <Trans>Collateral</Trans> ({collateralToken.symbol})
```

Comment:
let's move `({collateralToken.symbol})` inside <Trans></Trans>. e.g. in theory in some languages token symbol should be at the beginning or in the middle of the phrase